gradio/components.py

```diff
@@ -2184,7 +2207,9 @@ def process_single_file(f):
             )
             if self.type == "file":
                 if is_file:
-                    file = processing_utils.create_tmp_copy_of_file(file_name)
+                    temp_file_path = self.make_temp_copy_if_needed(file_name)
```

Comment:
The front-end can't fetch the input files of the zip_files demo

![zip_file_download](https://user-images.githubusercontent.com/41651716/205977360-f43185f5-d6b6-48ba-b4d4-08489b65c045.gif)

Reply:
When you don't cache the examples, you can't fetch the output or the input. When you do cache examples you can download the output. 

Here is the behavior without example caching

![zip_file_output_download](https://user-images.githubusercontent.com/41651716/205977882-1012069c-d310-4a55-9458-3940678a3795.gif)